components/Autocomplete.vue

```diff
@@ -70,10 +70,15 @@
 
 /* iteminp class is passed into this component in a prop */
 .iteminp ul {
+  /* 
   width: 100% !important;
   right: 0px !important;
-  padding-right: 15px !important;
-  padding-left: 15px !important;
+  margin-right: 15px !important;
+  margin-left: 15px !important;
+  */
+  /*Width adjusted for the border of the search options*/
```

Comment:
Lets try and keep comments consistent, IE space after the first * and before the last *